osidb/models/flaw/cvss.py

```diff
@@ -24,6 +25,12 @@ def create_cvss(flaw, issuer, version, **extra_fields):
             return FlawCVSS(flaw=flaw, issuer=issuer, version=version, **extra_fields)
 
 
+@pghistory.track(
+    pghistory.InsertEvent(),
+    pghistory.UpdateEvent(),
+    pghistory.DeleteEvent(),
+    model_name="FlawCVSSAudit",
+)
```

Comment:
There are two CVSS models, `FlawCVSS` and `AffectCVSS` since the Jira issue talks about enabling history for CVSS scores I get the feeling that `AffectCVSS` should be included too.